src/libraries/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.SymbolicLink.cs

```diff
@@ -112,7 +112,7 @@ public void FileSystemWatcher_SymbolicLink_IncludeSubdirectories_DoNotDereferenc
             using var dirA = new TempDirectory(GetTestFilePath());
             using var dirB = new TempDirectory(GetTestFilePath());
 
-            string linkPath = Path.Combine(dirA.Path, "linkToDirB");
+            string linkPath = Path.Combine(dirA.Path, Path.ChangeExtension("linkToDirB", ".link"));
```

Comment:
Is there any reason why it can't be just `linkToDirB.link`?

```suggestion
            string linkPath = Path.Combine(dirA.Path, "linkToDirB.link");
```